app/Http/Controllers/EmailController.php

```diff
@@ -33,4 +33,20 @@ public function update()
 
         return redirect()->route('emails')->with('flash_message', ['text' => 'Success: Email settings have been saved!', 'class' => 'alert-success']);
     }
+
+    public function testEmail()
+    {
+        return view('admin.settings.emails.test');
+    }
+
+    public function sendTestEmail()
+    {
+        // Build and send email
+        $email = new Email();
+        $input = Input::all();
+
+        $email->sendEmail('welcome', 'applicant', $input['recipient'], $input);
```

Comment:
I think we can just use this "welcome" email since it is in the database already but could also add a "test" email to the `emails` database too for this. I have no strong feelings which way assuming that we will always have a "welcome" email for each scholarship!

Reply:
@chloealee I think we'll have to make a new one? If we use the `welcome` email then it will use the welcome subject and body from the database I believe

Reply:
@chloealee and I don't think we need any `$extra_tokens` here so I don't think you need to pass a fourth parameter

Reply:
We could make the new email model within this function though, and then update/set the `subject` and `body`:
```
$email = Email::firstOrCreate(
    'key' => 'test',
    'recipient' => 'test',
);

$email->fill('subject' => $input['recipient'], 'body' => $input['body']);
$email->sendEmail($email->key, $email->recipient, $input['recipient']);
```

Reply:
oh this is brilliant, thanks @katie! updated :)